junifer/external/nilearn/junifer_nifti_spheres_masker.py

```diff
@@ -59,6 +64,155 @@
 """
 
 
+def _apply_mask_and_get_affinity(
```

Comment:
This is for adding the custom check of empty spheres right?

Reply:
Indeed the only change from nilearn was to remove the check.

Reply:
Sounds good.